Mod/Classes/Patched/MyOrbPickup.cs

```diff
@@ -0,0 +1,62 @@
+﻿using TowerFall;
+using Monocle;
+using Microsoft.Xna.Framework;
+using System;
+using Patcher;
+
+namespace Mod
+{
+	[Patch]
+	public class MyOrbPickup : OrbPickup 
+    {      
+		public MyOrbPickup(Vector2 position, Vector2 targetPosition, OrbTypes orbType) : base(position, targetPosition, orbType)
+        {
+			base.Tag(GameTags.PlayerGhostCollider);
+        }
+
+		public override void OnPlayerGhostCollide(PlayerGhost ghost)
+		{
+			base.Level.Layers[ghost.LayerIndex].Add(new LightFade().Init(this, null));
```

Comment:
Are you sure you can't call the method where this is happening instead of copying it? Or does that cause a crash because it calls Level.Add?